src/js/modules/edit.js

```diff
@@ -1601,7 +1601,7 @@ Edit.prototype.editors = {
 
 		function fillList(items, intialLoad){
 			var current = false;
-			var autoSelectIndex = -1;
+			var autoSelectIndex = editorParams.autoSelectFirstItem ? 0 : -1;
```

Comment:
Made a slightly more elegant solution that combines the addition and the bugfix (sorry for the multiple pushes)